examples/cartrade/package.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "@hyperledger-labs/cactus",
+  "private": true,
+  "scripts": {
+    "run-ci": "./tools/ci.sh",
```

Comment:
@takeutak  Does the cartrade example come with it's own CI script and lerna package structure embedded or is it just a matter of copy paste?